cmd/copy.go

```diff
@@ -688,13 +697,16 @@ func (cca *cookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 			BlockSizeInBytes:         cca.blockSize,
 			ContentType:              cca.contentType,
 			ContentEncoding:          cca.contentEncoding,
+			ContentLanguage:          cca.contentLanguage,
+			ContentDisposition:       cca.contentDisposition,
+			CacheControl:             cca.cacheControl,
```

Comment:
Unlike the two new fields above it, this one appears to be never used.  Presumably the fact that it's not used is a bug?

Reply:
CacheControl _does_ make it into the headers in my experience. Happy to add internal tests though.